README.md

```diff
@@ -89,9 +89,16 @@ The CLSI is based on a JSON API.
 }
 ```
 
+With `curl`, if you place the above json in a file called `data.json`, the request would look like this:
+
+``` shell
+$ curl -X POST -d @data.json localhost:3013/project/<id>/compile
+```
+
 You can specify any project-id in the URL, and the files and LaTeX environment will be persisted between requests.
 URLs will be downloaded and cached until provided with a more recent modified date.
 
+
```

Comment:
Nitpick - I don't think we need this blank line :-)

Reply:
I think this is a formatting habit I picked up from reading the Haiku OS source code. They had a rule to put two blank lines between major top-level elements.